src/eredis_cluster.erl

```diff
@@ -101,6 +101,50 @@ q(Command) ->
 qk(Command, PoolKey) ->
     query(Command, PoolKey).
 
+%% =============================================================================
+%% @doc Multi node query
+%% @end
+%% =============================================================================
+-spec qmn(redis_pipeline_command()) -> redis_pipeline_result().
+qmn(Commands) ->
+    CommandsByPools = split_by_pools(Commands),
```

Comment:
Yes, adrienmo, you are right in 
eredis_cluster:qmn([[<<"INCR">>,<<"foo">>],[<<"INCR">>,<<"foo">>]]).
I got [{ok,<<"1">>},{ok,<<"1">>}]
instead of [{ok,<<"1">>},{ok,<<"2">>}]